src/schema/associations.ts

```diff
@@ -86,6 +91,24 @@ const associationLookup = {
     }
   },
 }
+const multiAssociationLookup: Record<string, MultiLoadFunction> = {
```

Comment:
It was easier for me to wrangle the type system by adding a second lookup table than by trying to make `associationLookup` hold two type signatures.